src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,11 +2,27 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
-    private List<Actor> actors;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Using `FetchType.EAGER` for the `actors` list may lead to performance issues if the list is large, as it will load all actors associated with a movie whenever a movie is fetched. Consider using `FetchType.LAZY` to improve performance, unless eager fetching is specifically required.